modules/math/src/geospatial/helpers/scale-to-geodetic-surface.js

```diff
@@ -40,29 +40,33 @@ export default function scaleToGeodeticSurface(cartesian, ellipsoid, result) {
   const squaredNorm = x2 + y2 + z2;
   const ratio = Math.sqrt(1.0 / squaredNorm);
 
+  if (!Number.isFinite(ratio)) {
```

Comment:
When does this happen? Why did we have to add this? Add a comment?

Reply:
When very close to center, `[0, 0, 0]`, the value `ratio = Math.sqrt(1.0 / squaredNorm);` is infinite.